src/sentry/api/endpoints/organization_events_meta.py

```diff
@@ -1,14 +1,24 @@
 from __future__ import absolute_import
 
 import six
+from django.conf import settings
 
 from rest_framework.response import Response
 from rest_framework.exceptions import ParseError
 
+from sentry.api.base import EnvironmentMixin
 from sentry.api.bases import OrganizationEventsEndpointBase, OrganizationEventsError, NoProjects
+from sentry.api.helpers.group_index import build_query_params_from_request
+from sentry.api.event_search import parse_search_query
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.group import GroupSerializer
+from sentry.search.snuba.backend import EventsDatasetSnubaSearchBackend
 from sentry.snuba import discover
 
 
+IssueSeach = EventsDatasetSnubaSearchBackend(**settings.SENTRY_SEARCH_OPTIONS)
```

Comment:
Generally we don't build specific backends, and use the search proxy instead.

```python
from sentry import search
results = search.query(...)
```

I saw that this is also being done the way you have it in `OrganizationGroupIndex`. Perhaps @iProgramStuff knows why we're using a specific backend there?